examples/oed_utils.jl

```diff
@@ -237,6 +249,50 @@ function build_start_point(A, N, ub)
     return x, active_set, S
 end
 
+"""
+Build domain feasible for any node.
+"""
+function build_domain_point_function(domain_oracle, A, N, int_vars, initial_lb, initial_ub)
+    return function domain_point(local_bounds)
```

Comment:
as a convention, I think something like "find_indomain_point" is more readable for a function, having an action verb